tests/aws/services/cloudformation/v2/ported_from_v1/test_template_engine.validation.json

```diff
@@ -0,0 +1,107 @@
+{
+  "tests/aws/services/cloudformation/test_template_engine.py::TestImportValues::test_cfn_with_exports": {
```

Comment:
Did you just copy the validations over? These paths are wrong. I see the snapshots have not been updated as well. I don't mind not revalidating (though it would be nice!) but they should at least be consistent so revalidating in the future will change rather than add new validation items.

Reply:
Thanks, indeed these should be ported like in https://github.com/localstack/localstack/pull/12660